pkgs/development/compilers/ispc/default.nix

```diff
@@ -42,25 +40,26 @@ stdenv.mkDerivation rec {
 
   installPhase = ''
     mkdir -p $out/bin
-    cp ispc $out/bin
+    cp bin/ispc $out/bin
```

Comment:
this is making it fail for me
```
  cp: cannot stat 'bin/ispc': No such file or directory
cannot build derivation '/nix/store/02npn6n2amdq38j9ngfm1363n26kz2g0-embree-2.17.4.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/pcgvbqi8bkv7dpz7wj97clmm9b2h73ly-luxcorerender-2.0.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/fx5cdsq1rbmwjw510lhdhp49mva1dyzs-env.drv': 3 dependencies couldn't be built
[0 built (1 failed)]
error: build of '/nix/store/fx5cdsq1rbmwjw510lhdhp49mva1dyzs-env.drv' failed
https://github.com/NixOS/nixpkgs/pull/70851
3 package failed to build:
embree2 ispc luxcorerender
```

Reply:
actually, the default cmake phase makes this irrelevant. just remove the installPhase altogether